Add missing starknet assets

nbundi opened the pull request:
Adding missing Starknet assets to `coreAssets.json`

llamabutler commented:
The adapter at projects/vesu exports TVL: 
        
 
 
    starknet                  22.27 M
    starknet-borrowed         7.74 M
    borrowed                  7.74 M
    
    total                    22.27 M

dbejarano820 reviewed (approved):
Much needed, any other changes needed in the backend?

g1nt0ki commented on projects/helper/coreAssets.json:
```diff
@@ -1306,7 +1306,7 @@
     "WBTC": "0x03fe2b97c1fd336e750087d68b9b867997fd64a2661ff3ca5a7c771641e8e7ac",
     "ETH": "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
-    "WSTETH": "0x042b8f0484674ca266ac5d08e4ac6a3fe65bd3129795def2dca5c34ecc5f96d2",
+    "WSTETH_Legacy": "0x042b8f0484674ca266ac5d08e4ac6a3fe65bd3129795def2dca5c34ecc5f96d2",
```
please revert this, changing keys will have side effects on other projects using the same key

nbundi replied:
ok, so there now exist two wstETH tokens on starknet bc StarkGate, the canonical bridge, migrated the legacy one to a new implementation https://github.com/DefiLlama/DefiLlama-Adapters/pull/13558/files#diff-d2d52b699b64d1224d64e4f8017ab0bc41869eafd84c153530d5b0fdeb08f34cR1319

should I rename the new implementation something like "WSTETH_1"? if I do this is there a way to label the old one with something like "wstETH (legacy)" (which is not the official token symbol) and have the new implementation use the "wstETH" symbol? would be confusing otherwise.

nbundi replied:
nvm I fixed the keys

g1nt0ki reviewed (changes requested):
.

nbundi commented:
> Much needed, any other changes needed in the backend?

@dbejarano820 I'm not sure if there is any other change needed tbh. I added new starknet assets and updated the Vesu adapter accordingly. the numbers reported here lgtm https://github.com/DefiLlama/DefiLlama-Adapters/pull/13558#issuecomment-2664220618

llamabutler commented:
The adapter at projects/vesu exports TVL: 
        
 
 
    starknet                  21.71 M
    starknet-borrowed         7.62 M
    borrowed                  7.62 M
    
    total                    21.71 M
